### PR TITLE
Creating new branch for quickstart

### PR DESCRIPTION
**Pull request by fercobo-msft**
Creating for the quickstart.

### CODE REVIEW

File: concepts/data-connect-quickstart-02.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+1. While you are still signed in to the Microsoft 365 Admin Portal, select the **Settings > Org settings** menu item.
+
+1. Select the **Microsoft Graph Data Connect** service.
+
+    ![A screenshot showing the "Services" in the "Org settings" blade. A user is toggling on the Microsoft Graph Data Connect service in the Microsoft 365 admin center.](images/data-connect-m365-mgdc-toggle.png)
+
+1. Select the checkbox that says **turn Microsoft Graph Data Connect on or off for your entire organization** to enable Data Connect.
+
+    ![A screenshot showing the checkbox you have to tick in order to enable Data Connect for your entire organization.](images/data-connect-m365-enable-mgdc-for-org.png)
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-02.md:57]: This screenshot is huge! Can you resize? It takes up a lot of screen space when it's rendered on the site. You might even just cut off all of the leading text stuff and just grab the text of the checkbox.

File: concepts/data-connect-quickstart-06.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+<!-- markdownlint-disable MD002 MD041 -->
+
+A Microsoft 365 administrator has the ability to approve or deny consent requests. This can be done via the Microsoft 365 Admin Center or programmatically via PowerShell.
+
+## Approve consent requests via the Microsoft 365 Admin Center
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-06.md:5]: You might add prepend "Option 1:" to this heading (and "Option 2:" to the powershell one). Want to make it super-obvious that you only do one of these sections.

Or an even slicker option would be to make it a tabbed scenario, so the reader can pick one or the other. See https://review.docs.microsoft.com/en-us/help/contribute/tabbed-conceptual-overview?branch=master&tabs=windows%2Cazure-cli for the details. (Note: I'm not 100% sure that this works in a YML-based tutorial, but it's worth trying :D)

File: concepts/data-connect-quickstart-06.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        ```
+
+    1. Create a new Exchange Online PowerShell session and load (import) it.
+
+        ```powershell
+        $Session = New-PSSession -ConfigurationName Microsoft.Exchange -ConnectionUri https://ps.protection.outlook.com/powershell-liveid/ -Credential $UserCredential -Authentication Basic -AllowRedirection
+        Import-PSSession $Session -DisableNameChecking
+        ```
+
+        > [!IMPORTANT]
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-06.md:48]: Combine these. Two alerts back to back just looks odd.

File: concepts/data-connect-quickstart-07.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+1. Provide the **Azure Storage connection** a name of **AzureStorageConnectionString** and select **Next**.
+1. Select **Finish**.
+
+    ![A screenshot of the Visual Studio interface showing the Configure Azure Storage summary.](images/data-connect-vs-configure-sa-summary.png)
+
+## Create a new model class that will be used to store the email metrics
+
+1. In the  **Solution Explorer** tool window, right-click the **Models** folder and select **Add > Class**.
+
+    ![A screenshot of the Visual Studio interface showing how you can add a new class by right-clicking in the models folder.](images/data-connect-vs-add-new-model-class.png)
```
Comment (jasonjoh) [inline concepts/data-connect-quickstart-07.md:53]: Just a suggestion: There are multiple screenshots very similar to this (add class, add view, etc.). I think one is enough, the rest can just be the text instructions.

File: concepts/data-connect-quickstart-02.md (hunk 1 above)
Comment (fercobo-msft) [reply concepts/data-connect-quickstart-02.md:57]: Whoops! I didn't look like it that in my screen, let me change the screenshot! 👍🏼

Comment (fercobo-msft) [reply concepts/data-connect-quickstart-02.md:57]: Done!

File: concepts/data-connect-quickstart-06.md (hunk 2 above)
Comment (fercobo-msft) [reply concepts/data-connect-quickstart-06.md:5]: Definitely trying this out, @jasonjoh !!!!

File: concepts/data-connect-quickstart-06.md (hunk 3 above)
Comment (fercobo-msft) [reply concepts/data-connect-quickstart-06.md:48]: Changed, thanks for the feedback 😄

File: concepts/data-connect-quickstart-07.md (hunk 4 above)
Comment (fercobo-msft) [reply concepts/data-connect-quickstart-07.md:53]: Makes sense, I removed the subsequent images that were basically the same.

File: concepts/data-connect-quickstart-06.md (hunk 2 above)
Comment (fercobo-msft) [reply concepts/data-connect-quickstart-06.md:5]: Gonna push changes to see how it renders in the review site.

Comment (fercobo-msft) [reply concepts/data-connect-quickstart-06.md]: Looks amazing! Thanks, Jason!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**